UPGRADING.md

```diff
@@ -137,11 +137,11 @@ file.
 
 For example, considering this existing setting:
 ```toml
-minimum-gas-prices = "0.001uatone"
+minimum-gas-prices = "0.025uatone"
 ```
 Before upgrading, the setting should be changed to:
 ```toml
-minimum-gas-prices = "0.001uatone,0.001uphoton"
+minimum-gas-prices = "0.025uatone,0.001uphoton"
```

Comment:
Do we want to also put `0.025uphoton` for clarity ?

Reply:
There is already a PR for that, see #120. 
The gas price values need to be clarified and I opened a discussion in this PR specifically for that.

Reply:
my bad! closing this PR